charts/govuk-rails-app/templates/deployment.yaml

```diff
@@ -16,7 +16,7 @@ spec:
         app: {{ .Release.Name }}
     spec:
       securityContext:
-        runAsNonRoot: {{ .Values.securityContext.runAsNonRoot }}
```

Comment:
I would not make any changes to this yet until we tested - as if we set it to **true** now it will be applied everywhere this chart is used. suggest you leave this as it for now.